src/main/java/teammates/ui/controller/InstructorFeedbackAbstractAction.java

```diff
@@ -0,0 +1,164 @@
+package teammates.ui.controller;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import teammates.common.datatransfer.FeedbackSessionType;
+import teammates.common.datatransfer.attributes.CourseAttributes;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.EmailType;
+import teammates.common.util.SanitizationHelper;
+import teammates.common.util.TimeHelper;
+
+public abstract class InstructorFeedbackAbstractAction extends Action {
+
+    protected FeedbackSessionAttributes extractFeedbackSessionDataHelper(
+            FeedbackSessionAttributes newSession, List<String> sendRemainderEmailsList) {
+        return this.extractFeedbackSessionDataHelper(newSession, sendRemainderEmailsList);
+    }
```

Comment:
Did you mean to use polymorphism here to define some abstract method called extractFeedbackSessionDataHelper, and have it implemented by child classes? extractFeedbackSessionDataHelper is not a very good name, as it doesn't give any indication of what its doing.